sql/core/src/main/scala/org/apache/spark/sql/execution/datasources/DataSourceStrategy.scala

```diff
@@ -529,11 +532,14 @@ object DataSourceStrategy {
    * @param translatedFilterToExpr An optional map from leaf node filter expressions to its
    *                               translated [[Filter]]. The map is used for rebuilding
    *                               [[Expression]] from [[Filter]].
+   * @param nestedPredicatePushdownEnabled Whether nested predicate pushdown is enabled. Default is
+   *                                       disabled.
```

Comment:
What does `Default is disabled` means? we should add a default value in the argument like `nestedPredicatePushdownEnabled: Boolean = false`?

Reply:
Oh, forgot to change it. I was adding default value but removed it later.